data/remnant/remnant 2 cognizance.txt

```diff
@@ -1183,6 +1183,8 @@ mission "Remnant: Cognizance 25"
 		dialog "You have destroyed the ships that the Palavret left behind."
 	on waypoint
 		dialog "You have patrolled all six systems."
+	on visit
+		dialog "You have landed on <planet> but have not destroyed the ships the Palavret left behind. Better depart and go find them."
```

Comment:
This assumes that you know that those ships are there.
I suspect this message would also show if you return to Ssil Vida before visiting all 6 systems.
if explicitly pointing the ships is desired, I would suggest instead using an `on enter` for the system they are in pointing them out to the player (I suspect it is possible for the player to jump in and out without those ships targeting them if they are far enough away). That could also be used to set a condition that could perhaps be used in the `on visit` to decide whether or not the player should reasonably know that there are ships to destroy, or instead just hasn't visited every system yet.

Reply:
yeah that makes sense to me